frame-metadata/src/v16.rs

```diff
@@ -223,22 +217,25 @@ impl IntoPortable for ExtrinsicMetadata {
 	serde(bound(serialize = "T::Type: Serialize, T::String: Serialize"))
 )]
 pub struct TransactionExtensionMetadata<T: Form = MetaForm> {
+	/// Extension version.
+	pub version: u8,
```

Comment:
I've moved the version here, instead of having `	pub transaction_extensions: Vec<(u8, TransactionExtensionMetadata<T>)>.

Wdyt? // cc @jsdw @niklasad1  🙏